src/couchapps/WMStats/_attachments/js/WMStats.Utils.js

```diff
@@ -260,3 +260,20 @@ WMStats.Utils.formatStateTransition = function(stateTransition) {
   	return WMStats.Utils.expandFormat(stateTransition, "detail", transFunc);
   	
 };
+
+WMStats.Utils.entityMap = {
+    '&': '&amp;',
+    '<': '&lt;',
+    '>': '&gt;',
+    '"': '&quot;',
+    "'": '&#39;',
+    '/': '&#x2F;',
+    '`': '&#x60;',
+    '=': '&#x3D;'
+};
+
+WMStats.Utils.escapeHtml = function(string) {
+    return String(string).replace(/[&<>"'`=\/]/g, function (s) {
```

Comment:
I was going to say there is a close parenthesis missing here, but it's there in the very end.
What a horrible syntax!! I'm glad you fix all these javascript problems :)